### PR TITLE
Initialization

### PR DESCRIPTION
**Pull request by mpa-mxiang**
For this PR, I:

- Use React Router.
- Use React components.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,22 +11,18 @@ const App = () => (
       <nav>
         <ul>
           <li>
-            <Link to="/">Home</Link>
+            <Link to="/">Books</Link>
```
Comment (t-yanick) [inline src/App.js:14]: - [x] Great job working on this milestone. As per project requirements, in the components folder, you should have a Books component. At the moment that is missing. See screenshots below describing exactly what you need to add.
- Review guideline
image
- Project requirements
image

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @mpa-mxiang , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark: Routes correctly added
:heavy_check_mark:  Linter checks passing
:heavy_check_mark: Project should be initialised using [Create React App](https://create-react-app.dev/)

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Capitalising the first letter of your commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (omar-muhamad) [reply src/App.js:14]: - [ ] I think you misunderstood what the previous reviewer tried to say here, You only renamed the `Home` component to the `Books` component.
As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md#:~:text=/categories-,Create%20components%20inside%20of%20the%20/components%20directory,-%3A), Please do the following:
- Put the Books component inside the `src/components` directory and you also need to create 3 other components and render them inside the `Books` component:
1. `Book` component: Takes the book data as a prop and renders the title and the author and Has a button for removing books.
2. `BookList` component: Renders a list of books using the `Book` component.
3. `NewBookForm` component: Renders a form with inputs for the title and the author of the book and Has a button to submit.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @mpa-mxiang,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Kamba56 — APPROVED** [review]
Hi @mpa-mxiang, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] great job setting up project with react!👏👏👏
- [x] Great job adding books and categories components!👏👏👏
- [x] !👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**